plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryTypeUtils.java

```diff
@@ -19,6 +19,7 @@
 import io.trino.spi.TrinoException;
 import io.trino.spi.block.Block;
 import io.trino.spi.type.ArrayType;
+import io.trino.spi.type.DecimalType;
```

Comment:
Could you update "Trino type to BigQuery type mapping" section in `bigquery.rst`?

Reply:
updated